From 9169f2d0a96f0fc935d80c71e1c792193c1debc3 Mon Sep 17 00:00:00 2001
From: feeleep75 <mfpass@gmail.com>
Date: Fri, 21 Oct 2016 11:56:36 +0200
Subject: [PATCH] removing hardcoded nicehash

---
 nheqminer/main.cpp | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/nheqminer/main.cpp b/nheqminer/main.cpp
index a5d0769..da9aecc 100644
--- a/nheqminer/main.cpp
+++ b/nheqminer/main.cpp
@@ -33,8 +33,8 @@ void print_help()
 {
 	std::cout << "Parameters: " << std::endl;
 	std::cout << "\t-h\t\tPrint this help and quit" << std::endl;
-	std::cout << "\t-l [location]\tLocation (eu, usa, hk, jp)" << std::endl;
-	std::cout << "\t-u [username]\tUsername (bitcoinaddress)" << std::endl;
+	std::cout << "\t-l [location]\tStratum server:port" << std::endl;
+	std::cout << "\t-u [username]\tUsername (pool worker)" << std::endl;
 	std::cout << "\t-p [password]\tPassword (default: x)" << std::endl;
 	std::cout << "\t-t [num_thrds]\tNumber of threads (default: number of sys cores)" << std::endl;
 	std::cout << "\t-d [level]\tDebug print level (0 = print all, 5 = fatal only, default: 2)" << std::endl;
@@ -79,8 +79,8 @@ int main(int argc, char* argv[])
 	std::cout << "Special thanks to tromp for providing optimized CPU equihash solver" << std::endl;
 	std::cout << std::endl;
 
-	std::string location = "eu";
-	std::string user = "1DXnVXrTmcEd77Z6E4zGxkn7fGeHXSGDt1";
+	std::string location = "zec.coinmine.pl:7007";
+	std::string user = "yourworker.1";
 	std::string password = "x";
 	int num_threads = -1;
 	bool benchmark = false;
@@ -145,9 +145,10 @@ int main(int argc, char* argv[])
 
 
 	if (!benchmark)
-	{
-		std::string host = "equihash." + location + ".nicehash.com";
-		std::string port = "3357";
+	{	
+		size_t delim = location.find(':');
+	        std::string host = location.substr(0, delim);
+            	std::string port = location.substr(delim+1);
 
 		std::shared_ptr<boost::asio::io_service> io_service(new boost::asio::io_service);
 
-- 
2.10.1

