--- nheqminer/nheqminer/CMakeLists.txt
+++ nheqminer/nheqminer/CMakeLists.txt
@@ -117,11 +117,11 @@
     )
 #add_executable(${PROJECT_NAME} ${SRC_LIST})

-set(LIBS ${LIBS} ${Threads_LIBRARIES} ${Boost_LIBRARIES})
+set(LIBS ${LIBS} ${Boost_LIBRARIES})

 #message("-- CXXFLAGS: ${CMAKE_CXX_FLAGS}")
 #message("-- LIBS: ${LIBS}")

 add_executable(${PROJECT_NAME} ${SOURCE_FILES})
 #target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES})
-target_link_libraries(${PROJECT_NAME} ${LIBS})
+target_link_libraries(${PROJECT_NAME} ${LIBS} ${CMAKE_THREAD_LIBS_INIT})

